server-backend/src/SwapService.ts

```diff
@@ -77,12 +85,12 @@ export class SwapService implements OnApplicationBootstrap, OnApplicationShutdow
             assert(address);
             await this.nbxplorer.trackAddress(address);
         } else if (request.chain === 'LIQUID') {
-            const liquidNetwork = network === bitcoin ? liquid : liquidRegtest;
+            const liquidNetworkToUse = network === bitcoin ? liquidNetwork : liquidRegtest;
```

Comment:
this would cover `testnet` as well
```suggestion
            const liquidNetworkToUse = network === bitcoin ? liquidNetwork : network;
```

Reply:
I I didn't consider the testnet. Since it will be repeated a lot in the code, I'd rather create a new function that covers this case: `getLiquidNetwork`